src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -97,7 +97,7 @@ public boolean isActive() {
     }
 
     public List<OffenderCourtCase> getActiveCourtCases() {
-        return courtCases.stream().filter(OffenderCourtCase::isActive).collect(toUnmodifiableList());
+        return courtCases == null ? Collections.emptyList() : courtCases.stream().filter(OffenderCourtCase::isActive).collect(toUnmodifiableList());
```

Comment:
Can we not just a default on the court cases? e.g.

```
@Builder.Default
private List<OffenderCourtCase> courtCases = new ArrayList<>();
```

Reply:
Yep, that's a lot cleaner 👍